### ISSUE TITLE
Bug in Commentary section

### ISSUE THREAD

**Issue by jcowey** [papyri/sosol#132]
When in the editor, it is not possible to use the commentary section. By clicking on "Commentary" one lands at an HTTP ERROR 500.
Here is a live example:
http://dev.papyri.info/editor/publications/65822/dclp_text_identifiers/149357/edit
click on commentary
http://dev.papyri.info/editor/publications/65822/dclp_text_identifiers/149357/commentary
and the return is:
![bildschirmfoto 2019-02-07 um 15 34 24](https://user-images.githubusercontent.com/612992/52418238-e343d500-2aed-11e9-8e13-966be7e1db0f.png)

For the purpose of comparison I opened the same file in litpap.info
http://litpap.info/editor/publications/1736/dclp_text_identifiers/3578/edit
here clicking on 
http://litpap.info/editor/publications/1736/dclp_text_identifiers/3578/commentary
leads me to a page where I can "Add introductory matter" and use commentary
![bildschirmfoto 2019-02-07 um 15 36 47](https://user-images.githubusercontent.com/612992/52418385-3b7ad700-2aee-11e9-8ddb-2fd5764e8ba7.png)

**Comment by jcowey** [comment]
It would be great if the commentary section worked in dev.papyri.info

**Comment by ryanfb** [comment]
Getting `Java::JavaxXmlTransform::TransformerConfigurationException`: "Failed to compile stylesheet. 1 error detected."

Backtrace:
```
lib/jruby_xml.rb:425→ RUBY.get_transformer
--
lib/jruby_xml.rb:306→ RUBY.apply_xsl_transform
app/controllers/ddb_identifiers_controller.rb:154→ RUBY.commentary
```

**Comment by ryanfb** [comment]
Should be fixed by same fix for #133.
